config/sites/now.uiowa.edu/core.entity_form_display.node.article.minimal.yml

```diff
@@ -0,0 +1,124 @@
+uuid: 5736330e-e26f-4d7d-8979-7fe6e218747a
```

Comment:
Is there a reason this needs to be a complete split?

Reply:
i imagine that was just exported that way. i can try to switch it to a partial. i wonder why caused this one to do that when it is defined in the partial split section for this site.

Reply:
This was fixed in https://github.com/uiowa/uiowa/pull/7044/commits/530e4461c5b9d5518d6b8217d6024566a688d836